.github/workflows/new-cfe-notifications.yml

```diff
@@ -31,33 +31,46 @@ jobs:
     if: github.event.label.name == 'CFE Approved'
     runs-on: ubuntu-20.04
     steps:
-      - name: Extract PR URL from issue body
+      - name: Extract PR info from issue body
         id: extract-pr
         run: |
-          PR_URL=$(echo "${{ github.event.issue.body }}" | grep -oP 'https://github.com/[^/]+/[^/]+/pull/\d+')
+          BODY="${{ github.event.issue.body }}"
```

Comment:
seems like this renders issue body in shell, which is still unsafe. we should instead use one of [the recommended mitigation techniques](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#good-practices-for-mitigating-script-injection-attacks). I think passing the body in environment variable first and then using it would work well here.

Reply:
good point, made changes here: https://github.com/woocommerce/woocommerce/pull/52002/commits/f88df7ac30a02cf2f5fe23d86f34096969f2371e